test/common/testvm.py

```diff
@@ -107,6 +107,7 @@ def __init__(self, address=None, image=None, verbose=False, label=None, fetch=Tr
         self.arch = "x86_64"
 
         self.image = image or testinfra.DEFAULT_IMAGE
+        self.atomic_image = self.image in ["rhel-atomic", "fedora-atomic", "continuous-atomic"]
```

Comment:
maybe we can add the list of atomics in testinfra, then we'd have them all in the same place: https://github.com/cockpit-project/cockpit/blob/master/test/common/testinfra.py#L53